open_seq2seq/utils/utils.py

```diff
@@ -250,10 +250,16 @@ def log_summaries_from_dict(dict_to_log, output_dir, step):
   # the first call to this function
   sm_writer = tf.summary.FileWriterCache.get(output_dir)
   for tag, value in dict_to_log.items():
-    sm_writer.add_summary(
-      tf.Summary(value=[tf.Summary.Value(tag=tag, simple_value=value)]),
-      global_step=step,
-    )
+    if type(value) == type(tf.Summary.Value()):
```

Comment:
You should probably use isinstance here?

Reply:
And also can you mention in the documentation of this function or in the comment that now it can accept tf.Summary.Value's directly

Reply:
This is has been changed.